lms/djangoapps/grades/config/forms.py

```diff
@@ -0,0 +1,37 @@
+"""
+Defines a form for providing validation of subsection grade templates.
+"""
+import logging
+
+from django import forms
+
+from lms.djangoapps.grades.config.models import CoursePersistentGradesFlag
+
+from opaque_keys import InvalidKeyError
+from xmodule.modulestore.django import modulestore
+from opaque_keys.edx.locator import CourseLocator
+
+log = logging.getLogger(__name__)
+
+
+class CoursePersistentGradesAdminForm(forms.ModelForm):
```

Comment:
FUTURE: Seems like this could be a reusable component for other django apps as well.